src/views/ContinuationIn/ContinuationInReviewConfirm.vue

```diff
@@ -0,0 +1,305 @@
+<template>
+  <div id="continuation-in-review-confirm">
+    <!-- Review and Confirm -->
+    <section class="mt-10">
+      <header>
+        <h2>Review and Confirm</h2>
+        <p class="mt-4">
+          Review the information in your application. If you need to change or complete anything,
+          return to the step to make the necessary change.
+        </p>
+      </header>
+
+      <!-- Your Business in Home Jurisdiction -->
+      <v-card
+        id="your-business-in-home-jurisdiction-vcard"
+        flat
+        class="mt-6"
+      >
+        <CardHeader
+          icon="mdi-domain-plus"
+          label="Your Business in Home Jurisdiction"
+        />
+        <span class="pl-6">**TODO: Change this to the proper component</span>
+        <SummaryDefineCompany />
+      </v-card>
+
+      <!-- Your Business in BC -->
+      <v-card
+        id="your-business-in-bc-vcard"
+        flat
+        class="mt-6"
+      >
+        <CardHeader
+          icon="mdi-domain"
+          label="Your Business in BC"
+        />
+        <SummaryDefineCompany />
+      </v-card>
+
+      <!-- People and Roles -->
+      <v-card
+        id="people-and-roles-vcard"
+        flat
+        class="mt-6"
+      >
+        <CardHeader
+          icon="mdi-account-multiple-plus"
+          label="People and Roles"
+        />
+        <ListPeopleAndRoles :isSummary="true" />
+      </v-card>
+
+      <!-- Share Structure -->
+      <v-card
+        id="share-structure-vcard"
+        flat
+        class="mt-6"
+      >
+        <CardHeader
+          icon="mdi-sitemap"
+          label="Share Structure"
+        />
+        <ListShareClass
+          :isSummary="true"
+          :shareClasses="getCreateShareStructureStep.shareClasses"
+          :showErrorSummary="!getCreateShareStructureStep.valid"
+          :isContinuationInFiling="true"
+        />
+      </v-card>
+    </section>
+
+    <!-- Continuation Effective Date and Time -->
+    <section
+      id="continuation-effective-date-time-section"
+      class="mt-10"
+    >
+      <header>
+        <h2>Continuation Effective Date and Time</h2>
+        <p class="mt-4">
+          Select the Date and Time of incorporation for your business. You may select a date up
```

Comment:
Probably need to replace incorporation -> continuation in in here (a few instances).

Reply:
I've asked Yui about this Sev, he said the wording should be the same as incorporation application.